src/PcmPlayer.ts

```diff
@@ -66,7 +66,7 @@ export class PcmPlayer {
         channels: this.channels,
       },
     })
-    this.worklet.connect(this.context.destination)
+    this.worklet.connect(this.gainNode!)
```

Comment:
This was connecting the sink direct to the source, bypassing the gainNode

Reply:
Unsure if you would rather a check for this.gainNode rather than a non-null assertion?

Reply:
I would prefer failing fast if gainNode is not set via something like:

`  if (!this.gainNode) {
      throw Error(
        'Illegal state - gainNode does not exist - create a new PcmPlayer',
      )
    }
`